apps/taquito-test-dapp/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "taquito-test-dapp-vite",
```

Comment:
can we confirm whether changing the name doesn't have any implications on builds or deployments?

Reply:
deploying taquito-test-dapp currently failing, need to investigate if it's related to this name change?